integration-test/1972-big-parking-lots.py

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+import dsl
+from shapely.wkt import loads as wkt_loads
+from tilequeue.tile import deg2num
+from . import FixtureTest
+
+
+def _tile_centre(z, x, y):
+    from tilequeue.tile import num2deg
+    lat, lon = num2deg(x + 0.5, y + 0.5, z)
+    return (lon, lat)
+
+
+class BigParkingLotsTest(FixtureTest):
+
+    def test_osm_big_parking_capacity(self):
+        lon, lat = (8.9140471, 48.8352716)
+        self.generate_fixtures(
+            dsl.point(8712714905, (lon, lat), {
+                u"amenity": u"parking",
+                u"capacity": u"6472217472217",
+                u"parking": u"surface",
+                u"source": u"openstreetmap.org",
+            })
+        )
+
+        x, y = deg2num(lat, lon, 15)
+        self.assert_has_feature(
+            15, x, y, 'pois', {
```

Comment:
Change 15 to 16

Reply:
That did it, thanks! Fixed in 94e0755a.

Reply:
16 is the max zoom for zoom 256 tiles (while 15 is max for 512 tiles)